src/main/java/Leas_Liudmila/LLAdoptPage.java

```diff
@@ -40,6 +40,9 @@ public class LLAdoptPage {
     @FindBy(xpath = "//table/tbody/tr[1]")
     public WebElement firstAddedPet;
 
+    @FindBy(xpath = "//table/tbody/tr[2]")
+    public WebElement secondAddedPet;
```

Comment:
I think is no sense to create for 1-th , 2-th and so on separate variables
Please create a List<WebElement> for all rows